packages/backend/src/migration/1582983837547-users_table.ts

```diff
@@ -0,0 +1,14 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class usersTable1582983837547 implements MigrationInterface {
```

Comment:
błędna nazwa klasy

Reply:
To nie jest błąd, bo migrator podczas tworzenia nadaje sam nazwę

Reply:
obalone